roles/tpa_single_node/vars/main_example_nonaws.yml

```diff
@@ -59,15 +59,15 @@ tpa_single_node_oidc_provider_client_id: "{{ lookup('env', 'TPA_OIDC_PROVIDER_CL
 tpa_single_node_oidc_provider_client_secret: "{{ lookup('env', 'TPA_OIDC_PROVIDER_CLIENT_SECRET') | default('tpa_single_node_default_empty', true) }}"
 
 # TSL Certificates
-tpa_single_node_guac_csub_tls_cert_pem_path: "{{ tpa_single_node_certificates_dir }}/guac-collectsub-tls-certificate.pem"
-tpa_single_node_guac_csub_tls_cert_key_path: "{{ tpa_single_node_certificates_dir }}/guac-collectsub-tls-certificate.key"
-tpa_single_node_guac_graphql_tls_cert_pem_path: "{{ tpa_single_node_certificates_dir }}/guac-graphql-tls-certificate.pem"
-tpa_single_node_guac_graphql_tls_cert_key_path: "{{ tpa_single_node_certificates_dir }}/guac-graphql-tls-certificate.key"
-tpa_single_node_collector_osv_tls_cert_pem_path: "{{ tpa_single_node_certificates_dir }}/collector-osv-tls-certificate.pem"
-tpa_single_node_collector_osv_tls_cert_key_path: "{{ tpa_single_node_certificates_dir }}/collector-osv-tls-certificate.key"
+tpa_single_node_guac_csub_tls_crt_path: "{{ tpa_single_node_certificates_dir }}/guac-collectsub-tls-certificate.pem"
```

Comment:
Why the tpa_single_node_**_cert_pem_path are changed into *_tls_crt_path ? When the vars are changed the argument_specs.yml must be manually updated and the **make role-readme** executed to regenerate the roles/rhtpa_single_node/Readme-md

Reply:
@desmax74 because we're dealing with crt not pem.